OpenSim/Tools/AnalyzeTool.cpp

```diff
@@ -434,8 +442,14 @@ setStatesFromMotion(const SimTK::State& s, const Storage &aMotion, bool aInDegre
     _model->getSimbodyEngine().convertDegreesToRadians(*qStore);
     _model->getSimbodyEngine().convertDegreesToRadians(*uStore);
 
-    _statesStore = createStatesStorageFromCoordinatesAndSpeeds(*_model, *qStore, *uStore);
+    double ti = qStore->getFirstTime();
+    double tf = qStore->getLastTime();
+    uStore->addToRdStorage(*qStore, ti, tf);
+
+    delete _statesStore;
+    _statesStore = new Storage(512, "states");
 
+    _model->formStateStorage(*qStore, *_statesStore);
     delete qStore;
```

Comment:
Agree the removal can wait, but would be good to add some documentation to the header file warning against use or the pitfalls in case we don't get around to complete removal.